venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -194,7 +118,7 @@ function() {
           signatures: [CONF],
           writers: [CONF],
           invitees: ['~'],
-          noninvitees: [authorGroupId, reviewerGroupId, areachairGroupId],
```

Comment:
you can keep the reviewerGroupId and areachairGroupId, it doesn't matter if they don't exist.